mogp_emulator/tests/test_fitting.py

```diff
@@ -71,13 +71,13 @@ def test_fit_GP_MAP_MOGP():
 
     theta_exp = np.zeros((2, 3))
     theta_exp[0] = np.array([ 1.6030532031342832, -2.090511425471982 , -0.7803960307137198])
-    theta_exp[1] = np.array([ 1.2807839212942222, -0.5095550757153631, -2.009987161484929 ])
+    theta_exp[1] = np.array([ 1.414112951818647 , -0.5974688573393576,  0.6857536842773265])
     logpost_exp = np.zeros(2)
     logpost_exp[0] = -296.0297245831661
-    logpost_exp[1] = -249.82550942156362
+    logpost_exp[1] = -250.06025683867367
```

Comment:
Why did these change?

Reply:
I changed these to make the test run on a single process to test that the single thread version gives the same results as the old version (and to ensure general consistency across systems). The lack of multiple threads changed the sequence of random seeds, meaning that the second emulator started from a different point, so this corrects that. This is really a regression test, so the numbers don't really mean anything, just that we get consistent results when starting from the same point.